packages/contracts-bedrock/contracts/legacy/AddressResolver.sol

```diff
@@ -1,40 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.9;
-
-/* Library Imports */
-import { AddressManager } from "./AddressManager.sol";
-
-/**
- * @title AddressResolver
- */
-abstract contract AddressResolver {
```

Comment:
When did this cease to be in use?

Reply:
It was never in use for contracts-bedrock, not sure why it was copied in the first place